code/modules/unit_tests/_unit_tests.dm

```diff
@@ -40,6 +49,11 @@
 #define TEST_DEFAULT 1
 #define TEST_DEL_WORLD INFINITY
 
+/// Change color to red on ANSI terminal output. As unit tests are usually output to consoles, this should be fine.
```

Comment:
i'll remember this for when it goes wrong

Reply:
Note since this is probably the big question you have regarding that, it just looks ugly if forcibly run in the Windows DreamDaemon rather than actually breaking it (which as far as I'm aware we don't support doing in the first place, but useful for testing the unit tests themselves). The tests.log also looks similar, but again is an artifact that's usually inaccessible outside of manipulating the unit test system locally.

![image](https://user-images.githubusercontent.com/1185434/166177173-59a6f426-2236-49f3-903c-1ece9ce5d89a.png)

Reply:
I use dream daemon for tests all the time, can you make this a define instead? That way we can add it through `-D` in CI.

Reply:
Gotcha, that's reasonable

Reply:
Done.